### DISCUSSION TITLE
半杯珍奶的錢就能買到一個人的信用卡資訊

### DISCUSSION THREAD

**Discussion by a3240281370** [1101-GS4719-C/Python-Community-GS4719-C#1410] [General]
不知道大家以前升學時有沒有類似的經驗，莫名奇妙就接到補習班電話問要不要來上課，對方甚至對自己的考試成績、家庭成員超熟悉
根據統計，全球在過去十年之間共有超過 40 億起個人資料被盜或外洩的紀錄而光是去年，就有 2300 萬筆的台灣人個資（比身分證更詳細）在國外網站上大甩賣
也正因為企業蒐羅個資宛如探囊取物，又不必負擔保管責任、交代後續使用情況，使用者的資訊安全其實暴露在極大風險中，除了剛剛提到的補習班，大家可能也都有過被車貸飆股騷擾、被詐騙被盜刷的經驗
希望政府能更加重視資安方面的問題！

**Comment by sjh501** [comment]
我覺得除了政府方面，民眾在個資上的觀念也要加強。像我周遭的人很多覺得自己不是什麼大人物，個資外流也不會有什麼影響；大多數的人在各種同意書的同意、簽署上也會因為感覺很麻煩而跳過，直接勾選同意或簽署，沒有確認企業對於個資的使用範圍和權限。這些習慣其實都直接或間接的造成個資外流的問題。
